src/index.php

```diff
@@ -0,0 +1,356 @@
+            <div class="shareButton">
+              <input type="checkbox" id="share" value="share" name="share" />
+              <label for="share">twitterにシェアする</label>
+            </div>
+          </div>
+        </div>
+      </div>
+      <a href="" target="blank" id="link_share">
+        <button class="button-design button-modal js-record">
+          記録・投稿
+        </button>
+      </a>
+    </div>
+  </div>
+
+  <!-- ここからtop -->
+  <header>
+    <div><img src="./img/POSSElogo.jpg" alt="" /></div>
+    <div class="now_week">4th week</div>
+    <button onclick="showModal()" class="js-open button-design button-pc">
+      記録・投稿
+    </button>
+  </header>
+  <main>
+    <div class="container">
+      <div class="inner-container">
+        <div class="left-container">
+          <div class="timer">
+            <div>
+              <p class="timer_title">Today</p>
+              <p class="timer_number"><?= $Today ?></p>
+              <p class="timer_hour">hour</p>
+            </div>
+            <div>
+              <p class="timer_title">Month</p>
+              <p class="timer_number"><?= $Month ?></p>
+              <p class="timer_hour">hour</p>
+            </div>
+            <div>
+              <p class="timer_title">Total</p>
+              <p class="timer_number"><?= $Total ?></p>
+              <p class="timer_hour">hour</p>
+            </div>
+          </div>
+          <div class="bar_graph">
+            <div id="chart_div" style="width: 100%"></div>
+          </div>
+        </div>
+        <div class="right-container">
+          <div class="donuts" alt="">
+            <div class="study_contents">
+              <p>学習コンテンツ</p>
+              <div class="chartBox">
+                <!-- (仮クラス名) -->
+                <div class="chart-container">
+                  <div id="study_contents_chart" class="chart"></div>
+                </div>
+                <div class="regend study_contents_chart_regend">
+                  <ul>
+                    <li><span></span>ドットインストール</li>
+                    <li><span></span>N予備校</li>
+                    <li><span></span>POSSE課題</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div class="study_language">
+              <p>学習言語</p>
+              <div class="chartBox">
+                <div class="chart-container">
+                  <div id="study_language_chart" class="chart"></div>
+                </div>
+                <div class="regend study_language_chart_regend">
+                  <ul>
+                    <li><span></span>HTML</li>
+                    <li><span></span>CSS</li>
+                    <li><span></span>JavaScript</li>
+                    <li><span></span>PHP</li>
+                    <li><span></span>Laravel</li>
+                    <li><span></span>SQL</li>
+                    <li><span></span>SHELL</li>
+                    <li><span></span>情報システム基礎知識（その他）</li>
```

Comment:
連続する系はforeach使ってあげると要素が増えた時に追加忘れもなくなり良いと思います
```suggestion
                    <?php foreach ($ZZZs as $ZZZ) { ?>
                        <li><span></span><?= $ZZZ ?></li>
                    <?php } ?>
```